src/components/CountryList.vue

```diff
@@ -10,6 +10,7 @@ import CountryBoard from "@/components/CountryBoard.vue";
 
 <style lang="scss" scoped>
 .v-countries-list {
+  height: 100vh;
```

Comment:
Czemu height, a nie `minHeight`?

Reply:
Sprawdzałeś responsywność?

Reply:
min-height jest bezpieczniejsze, jak nadasz teraz overflow:hidden na ten kontener to się zdziwisz :D

Reply:
Zgadza się, `min-height` jest bezpieczniejsze. Dlaczego?
Dlatego, że jak dam overflow hidden to komponent z flagami przyjmie tylko wysokość taką na jaką pozwala mu rodzic. Te elementy się schowają, a strona nie pozwoli scrollowac po wszystkich krajach tylko wyswietli te ktore zmieszczą się w wysokości 100vh. Dobra rada